src/script/lua_api/l_client.cpp

```diff
@@ -169,6 +174,11 @@ int ModApiClient::l_get_node(lua_State *L)
 {
 	// pos
 	v3s16 pos = read_v3s16(L, 1);
+
+	if (getClient(L)->getCSMFlavourLimits() & CSMFlavourLimit::CSM_FL_LOOKUP_NODES) {
+		// TODO
```

Comment:
A compromise solution IMO would be to make the pointed node always gettable, so that although LOOKUP_NODES is disabled, CSM mods still can display information on the pointed node. This should be accomplished by an extra callback while this just returns nil.